src/main/drivers/accgyro_mpu3050.c

```diff
@@ -23,26 +23,20 @@
 #include "common/maths.h"
 
 #include "system.h"
+#include "exti.h"
 #include "bus_i2c.h"
 
 #include "sensor.h"
 #include "accgyro.h"
+#include "accgyro_mpu.h"
 #include "accgyro_mpu3050.h"
 
-
+extern mpuDetectionResult_t mpuDetectionResult;
```

Comment:
yup. there is much cleanup work still to be done on this code, i ran out of time on saturday and just pushed what I had working.